_pages/en_US/troubleshooting.txt

```diff
@@ -118,7 +118,7 @@ If neither of these solutions fixes this problem, then custom firmware may have
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>DSiWare Management menu does not crash</u></summary>
+<summary><u>DSiWare Management menu does not crash or displays "No accessible software data."</u></summary>
```

Comment:
I think we can just straight up remove the "does not crash" section. It will either crash, show a question mark, or show nothing.

Will leave it up to PR author to decide on that.

Reply:
by section I mean just the title name, don't remove the whole troubleshooting lol

i.e. `s/does not crash//`

Reply:
I would suggest "and" instead of "or"